frontends/election-manager/src/components/hand_marked_paper_ballot.tsx

```diff
@@ -607,22 +609,33 @@ export function HandMarkedPaperBallot({
       localeElection.ballotStrings
     );
   }
-  const primaryPartyName = getPartyFullNameFromBallotStyle({
-    ballotStyleId,
-    election,
-  });
+  const primaryPartyName = !isSuperBallotStyle(ballotStyleId)
+    ? getPartyFullNameFromBallotStyle({
+        ballotStyleId,
+        election,
+      })
+    : undefined;
```

Comment:
I know all the added conditioning in this file isn't great. After m14, it's definitely worth cleaning up; it just felt too high risk to move significant chunks of code around before m14, especially when HMPBs don't have test coverage. I believe because of their post-processing, they're hard to test and we've accordingly mocked them:

https://github.com/votingworks/vxsuite/blob/db44e38e82e9a534ba227a643000a46b4dd85e62/frontends/election-manager/src/app.test.tsx#L74

Reply:
good compromise.